app/templates/device-search-results.html

```diff
@@ -46,6 +46,7 @@ <h1>{{ page_title_value }}</h1>
 				'previous_device_id':previous_device_id ,
 				'previous_field_device_phone_number': previous_field_device_phone_number,
 				'previous_ons_id': previous_ons_id,
+				'previous_employee_id': previous_employee_id,
```

Comment:
The indentation here seems to be off! It may have inserted tabs instead of spaces.